docs/tutorials/fundamentals/quick_start.md

```diff
@@ -58,8 +58,8 @@ You will also see some examples of plugins. The core napari viewer focuses on do
 
 - For those familiar with Python:
 
-    napari can be installed on most macOS, Linux, and Windows systems with Python 3.7, 3.8, and 3.9 using pip. 
-    
+    napari can be installed on most macOS, Linux, and Windows systems with Python 3.7, 3.8, and 3.9 using pip.
```

Comment:
for another PR... but just noting that we need to update the tutorials to say py 3.8-3.10